src/App.tsx

```diff
@@ -1,16 +1,25 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const addMovie = (movie: Movie) => {
+    setMovies([...movies, movie]);
```

Comment:
good practice to use a callback when we need a prevState
```suggestion
    setMovies((prevMovies) => [...prevMovies, movie]);
```